src/lib/markets/chain.ts

```diff
@@ -359,10 +370,19 @@ async function fetchMarketSnapshotsAfterSettle({
     ],
   })
 
+  const events = await simulateMarketSettles({
```

Comment:
Can we use this simulate to also get the return value of the lens call so we don't need to do the additional `simulateContract` above? Otherwise we're effectively doubling the number of JSONRPC calls for our most heavily used query

Reply:
Ive re-used the result and decoded it into the original function result we had